### PR TITLE
parser(fields): add more fields to the parser

### PR DESCRIPTION
**Pull request by shdwkl**
bigauto -> BigAutoField,
bigint -> BigIntegerField,
binary -> BinaryField,
decimal -> DecimalField,
duration -> DurationField,
genericip -> GenericIPAddressField,
image -> ImageField,
nullbool -> NullBooleanField,
positiveint -> PositiveIntegerField,
positivesmallint -> PositiveSmallIntegerField,
smallint -> SmallIntegerField,

### CODE REVIEW

**Comment by mrswats** [comment]
I have to fix something with the development dependencies, but thanks for the PR!

I think I'm missing some more tests, too.

**Comment by shdwkl** [comment]
> development dependencies

already solved in you last PR but you haven't merge it yet, removing hashes or you could solve it by editing 
`pyproject.toml` 
```toml
 dependencies = [
   - "Django>=3.2.15"
   + "Django==4.1.2"
 ]
```


with

**Comment by shdwkl** [comment]
> some more test

these 3 tests fails because `all_raw_fileds` list actually doesn't contains all raw fields.

File: tests/test_parse_fields.py [hunk 1]
```diff
@@ -6,25 +6,38 @@
...
+    "int : int",
+    "nullbool : nullbool",
+    "positiveint : positiveint",
+    "positivesmallint : positivesmallint",
+    "slug : slug",
+    "smallint : smallint",
+    "text : text",
+    "time : time",
+    "url : url",
+    "uuid : uuid",
```
Comment (mrswats) [inline tests/test_parse_fields.py:34]: There's no need to add spaces between `:` which in turn adds more stuff into the parsing logic.

**Comment by mrswats** [comment]
I decided I don't want this, for now. I want to keep the tool simple. If you need more advanced fields like the ones you added, the user can add them manually.

Thanks for the contribution none the less.

**Pull request closed without merging**